src/main/kotlin/org/taktik/icure/be/ehealth/logic/kmehr/smf/impl/v23g/SoftwareMedicalFileImport.kt

```diff
@@ -788,13 +801,23 @@ class SoftwareMedicalFileImport(val patientLogic: PatientLogic,
                             )
 
                         },
-                "Commentaire" to  Content().apply {stringValue= item.texts.joinToString(" ") { it.value } }
+                "Commentaire" to  Content().apply {stringValue= item.texts.joinToString(" ") { it.value } },
+                "pourcentage" to
+                        item.contents.find { it.incapacity != null }?.let {
+                            it.incapacity.percentage
+                        }?.let {
+                            Content().apply { numberValue = it.toDouble() }
+                        },
+                "Sortie" to
+                        item.contents.find { it.incapacity != null }?.let {
+                            it.incapacity.isOutofhomeallowed
+                        }?.let {
+                            Content().apply { stringValue = if (it) "allowed" else "forbidden" }
```

Comment:
Use a ternary operator

Reply:
It doesn't seem to be a feature in Kotlin.